src/stylesheets/core/mixins/_layout-grid.scss

```diff
@@ -42,7 +42,7 @@
     @elseif map-has-key($uswds-column-gaps, $gap) {
```

Comment:
@thisisdano shouldn't this be `else if` not `elseif`? http://sass-lang.com/documentation/file.SASS_REFERENCE.html#control_directives__expressions

Reply:
Yeah, I've been using `esleifs`, but I can find change to `else if`

Reply:
@thisisdano in this PR or a separate one?